src/Accounts/Accounts/help/Add-AzEnvironment.md

```diff
@@ -44,6 +44,12 @@ Add-AzEnvironment [-Name] <String> [[-StorageEndpoint] <String>] [-ARMEndpoint]
  [<CommonParameters>]
 ```
 
+### Discovery
+```
+Add-AzEnvironment [-AutoDiscover] [-Uri <Uri>] [-Scope <ContextModificationScope>]
```

Comment:
We should add an example for the parmeter set

Reply:
I'd like to provide example once official DNS is enabled. Currently, no official link supports it